### DISCUSSION TITLE
Any info on a timeline for Actions GA?

### DISCUSSION THREAD

**Discussion by fernap3** [community/community#25861] [Actions]
<p>I’ve been loving GitHub actions so far and although it’s early, I am far enough along in a proof of concept to believe that we’ll want to switch my organization’s build system over to GitHub.  Is there any tentative timeline for when Actions will be endorsed for use in production scenarios?</p>
<p>Thanks!</p>
<p>Peter</p>

**Comment by chrispat** [comment, marked as answer]
<p>We announced at the beta launch in August that Actions will GA on November 13th at GitHub Universe.</p>
